var/spack/repos/builtin/packages/arbor/package.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Arbor(CMakePackage):
+    """Arbor is a high-performance library for computational neuroscience
+    simulations."""
+
+    homepage = "https://github.com/arbor-sim/arbor/"
+    url      = "https://github.com/arbor-sim/arbor/archive/v0.2.tar.gz"
+
+    version('0.2', git='https://github.com/arbor-sim/arbor.git',
+            commit='6f70bfee22415fc8cca5f12ab7f3917a043bddb3',
+            submodules=True)
```

Comment:
The URL given above works.  So no need to do a git-type download.  Please change to:
```
version('0.2', sha256='...')
```

Reply:
I can merge after this issue is resolved.

Reply:
actually I prefer the git solution and it is explicitly pointed out in the documentation